conformance/utils/http/http.go

```diff
@@ -51,6 +51,9 @@ type ExpectedResponse struct {
 	Backend   string
 	Namespace string
 
+	// MirroredTo is the destination pod of the mirrored request.
+	MirroredTo string
```

Comment:
How would we know a specific Pod? I think most of our Services have at least 2 Pods. We could theoretically change one of our base services to only have 1 Pod to simplify testing this, but in any case this would probably need to be a Service name.

Reply:
I read logs from all pods with a label `app: <tc.MirroredTo value>`. (which is similar to how the service pick the pods it expose) 

I could maybe query the service and find which pods are attached to it if you think thats better.

Reply:
We could also add a service with 1 Pod to the base manifests to make it clearer.

Reply:
I am gonna resolve it but feel free to open it again in case needed